tools/src/main/scala/scala/scalanative/build/GC.scala

```diff
@@ -32,6 +33,9 @@ object GC {
   /** Mostly-precise mark-region garbage collector. */
   def immix: GC = Immix
 
+  /** Mostly-precise mark-region garbage collector running concurrently. */
+  def commix: GC = Immix
```

Comment:
Should it be `Commix` instead of `Immix` here ?

Reply:
You are right.